guides/common/modules/proc_making-the-custom-discovery-iso-bootable.adoc

```diff
@@ -0,0 +1,29 @@
+[id="making-the-custom-discovery-iso-bootable_{context}"]
+= Making the custom Discovery ISO bootable
```

Comment:
Do you really need a separate procedure module for converting an iso file, installing grub2, and adding a checksum? I recommend combining this with an existing procedure module.

Reply:
I can actually do that now.